cms/templates/js/publish-xblock.underscore

```diff
@@ -101,6 +101,10 @@ var visibleToStaffOnly = visibilityState === 'staff_only';
             </a>
             </li>
         </ul>
+        <p>
+            <%- gettext("Note: Do not hide graded assignments using this feature.") %>
```

Comment:
@efischer19 Based on this dev work, I'm going to rework existing sections in the course authors' guide, so I'll provide a link when I have the new structure, and will also suggest link text.